<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-N46G1GJ" failures="5" tests="6" name="Test-2" time="144.461" errors="0" timestamp="2024-12-24T12:54:21 IST">
  <testcase classname="com.crm.base.contact.ContactTest" name="createCon" time="22.667"/>
  <testcase classname="com.crm.base.contact.ContactTest" name="createConWithOrg" time="5.905">
    <failure type="java.lang.AssertionError" message="expected [Administrator - Contacts - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]">
      <![CDATA[java.lang.AssertionError: expected [Administrator - Contacts - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]
at com.crm.base.contact.ContactTest.createConWithOrg(ContactTest.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- createConWithOrg -->
  <testcase classname="com.crm.base.contact.ContactTest" name="createConWithOrgAndDate" time="5.911">
    <failure type="java.lang.AssertionError" message="expected [Administrator - Contacts - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]">
      <![CDATA[java.lang.AssertionError: expected [Administrator - Contacts - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]
at com.crm.base.contact.ContactTest.createConWithOrgAndDate(ContactTest.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- createConWithOrgAndDate -->
  <testcase classname="com.crm.base.organization.OrganizationTest" name="createOrg" time="5.463">
    <failure type="java.lang.AssertionError" message="expected [Administrator - Organizations - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]">
      <![CDATA[java.lang.AssertionError: expected [Administrator - Organizations - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]
at com.crm.base.organization.OrganizationTest.createOrg(OrganizationTest.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- createOrg -->
  <testcase classname="com.crm.base.organization.OrganizationTest" name="createOrgWithIndustry" time="5.757">
    <failure type="java.lang.AssertionError" message="expected [Administrator - Organizations - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]">
      <![CDATA[java.lang.AssertionError: expected [Administrator - Organizations - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]
at com.crm.base.organization.OrganizationTest.createOrgWithIndustry(OrganizationTest.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- createOrgWithIndustry -->
  <testcase classname="com.crm.base.organization.OrganizationTest" name="createOrgWithPhoneNumber" time="5.646">
    <failure type="java.lang.AssertionError" message="expected [Administrator - Organizations - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]">
      <![CDATA[java.lang.AssertionError: expected [Administrator - Organizations - vtiger CRM 5 - Commercial Open Source CRM] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]
at com.crm.base.organization.OrganizationTest.createOrgWithPhoneNumber(OrganizationTest.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- createOrgWithPhoneNumber -->
  <testcase classname="com.crm.generic.baseclassutility.BaseClass" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase classname="com.crm.generic.baseclassutility.BaseClass" name="@BeforeSuite beforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
</testsuite> <!-- Test-2 -->
